README.md

```diff
@@ -1,8 +1,62 @@
+
+## Skills
+
+**Proficient:** Java, Algorithms & Data Structures, Linux/Bash, Agile/Scrum, Git
+
+**Competent:** C, JavaScript, RDBMS/SQL, HTML/CSS, React Native, LaTeX
+
+**Familiar:** TypeScript, React, Python, JUnit, MATLAB, UML
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring Agent which enables users to monitor the individual Linux hosts in the system. PostgreSQL database provisioned using Docker. Bash scripts programmed to extract hardware specifications and, with the use of crontab, periodically update the database with usage data.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/core_java)]:
+      
+  - Twitter App: ...
+  - JDBC App: ...
+  - Grep App: ...
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_PrinceSandhu/tree/master/cloud_devops)]: Not Started
+
+
+## Highlighted Projects
+**Project Emacs (Year IV Capstone Project)**: Developed an application utilizing Typescript, HTML, and CSS, while adhering to the software development life cycle, that enables users to take textual and handwritten notes while organizing and visualizing them in a non-hierarchical convention, received an A+.
+
+**GlobeTrans (Year III Cornerstone Project)**: Designed a large software system compliant to the specified business events and user scenarios, in an agile and team environment; received an A+. Design process included: establishing requirements, identifying, implementing, and documenting the architectural and detailed designs, developing and testing the prototype.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (December 2021-present)**: Exercised agile methodologies on a daily basis in a team environment to design and implement numerous software projects using a vide variety of technologies such as Linux, PostgreSQL, Docker, and Git.
```

Comment:
Exercised agile methodologies on a daily basis in a team environment to design and implement numerous software projects using a wide variety of technologies such as Linux, PostgreSQL, Docker, and Git.

fix typo and reword